src/lib/select/select.ts

```diff
@@ -299,7 +299,10 @@ export class MatSelect extends _MatSelectMixinBase implements AfterContentInit,
   private _disableOptionCentering: boolean = false;
 
   /** Whether the select is focused. */
-  focused: boolean = false;
+  get focused(): boolean {
```

Comment:
A setter needs to be defined here as well.  

With this change, the API is changed as `focused` becomes readonly.

Reply:
I didn’t add a setter, because the MatFormFieldControl interface calls for the property to be readonly. That being said, it doesn’t look like TS actually enforces it.

Reply:
Hmm okay, Lets add the setter as deprecated and then we can move it to readonly later